src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -460,6 +460,26 @@ static Library()
                     returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
                     targetFunction: SingleArgTrig(x => x * 180.0 / Math.PI))
             },
+            {
+                BuiltinFunctionsCore.Dec2Hex,
+                StandardErrorHandling<NumberValue>(
+                    expandArguments: InsertDefaultValues(outputArgsCount: 2, fillWith: new BlankValue(IRContext.NotInSource(FormulaType.Blank))),
+                    replaceBlankValues: ReplaceBlankWithZero,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<NumberValue>,
+                    checkRuntimeValues: FiniteChecker,
+                    returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
+                    targetFunction: Dec2Hex)
+            },
+            {
+                BuiltinFunctionsCore.Dec2HexT,
```

Comment:
>Dec2HexT [](http://example.com/codeflow?start=37&length=8)

Tabular version of scalar functions should go in the SimpleFunctionTabularOverloadImplementations section below